cmd/rb-main.go

```diff
@@ -235,7 +235,10 @@ func mainRemoveBucket(cliCtx *cli.Context) error {
 		}
 
 		listCtx, listCancel := context.WithCancel(ctx)
-		for range clnt.List(listCtx, opts) {
+		for obj := range clnt.List(listCtx, opts) {
+			if obj.Err != nil {
+				continue
```

Comment:
Good catch, but there is no point continuing here: if we are going to delete a bucket, we should list all of its contents and remove them. fatalIf() is enough.

Reply:
continue is to look for situations if there are objects @vadmeste such that we don't prematurely send DeleteBucket()